wordpress-mobile/WordPress-Android
# Update Sentry SDK to 2.1.3

This PR updates the Sentry SDK to enable the new release health feature. For the migration, I have followed the guide at https://docs.sentry.io/platforms/android/migrate/. Since Sentry no longer uses `sentry.properties` file, I've removed that from the mobile-secrets repo and updated the `gradle.properties` with the new credentials and applied this change in 7e46006fc0e8ac1860dd638bf8f28937d1d5bd0e.

_P.S: Thank you to @jkmassel who figured out that I was using `Sentry.init` instead of `SentryAndroid.init` and wasting my time testing the integration - and to @loremattei who reminded me to remove `sentry.properties` from the `.configure` file. Did I even do anything myself in this PR? 🤕_

**To test:**
* Add the following to `PostsListActivity.onCreate`: (you can change the message)
```
try {
    throw IllegalStateException("Testing new Sentry integration, if you see this crash as an open issue, please let Oguz Kocer know")
} catch (e: Exception) {
    CrashLoggingUtils.log(e)
}
```
* Run the app and open the post list
* Verify that Sentry issue shows up. You can search for the issue with the following query `postslistactivity error.type:IllegalStateException `

**PR submission checklist:**
